.github/CODEOWNERS

```diff
@@ -65,7 +65,7 @@ app/controllers/v0/feature_toggles_controller.rb @department-of-veterans-affairs
 app/controllers/v0/form1010_ezr_attachments_controller.rb @department-of-veterans-affairs/vfs-10-10 @department-of-veterans-affairs/backend-review-group
 app/controllers/v0/form1010_ezrs_controller.rb @department-of-veterans-affairs/vfs-10-10 @department-of-veterans-affairs/backend-review-group
 app/controllers/v0/form1010cg/attachments_controller.rb @department-of-veterans-affairs/vfs-10-10 @department-of-veterans-affairs/backend-review-group
-app/controllers/v0/form1095_bs_controller.rb @department-of-veterans-affairs/vfs-1095-b @department-of-veterans-affairs/backend-review-group
```

Comment:
Several of the changes in this file, including this one, are for changing 1095b file ownership to IIR. This probably could have been done in a separate ticket.

Reply:
👍 lgtm. https://github.com/orgs/department-of-veterans-affairs/teams/vfs-1095-b doesn't really have anyone in it anymore.